data/mods/Xedra_Evolved/mutations/paraclesians/homullus_mutations.json

```diff
@@ -0,0 +1,363 @@
+    "category": [ "HOMULLUS" ],
+    "threshreq": [ "THRESH_HOMULLUS" ],
+    "//": "Build an EOC finds where you are inside, outside, in city, in wilderness, above ground or below ground and based on where you are something horrificly damaging occurs to yoru enemy.  In a city maybe a piano falls on them, in the wilderness maybe a meteor crashes onto their head.",
+    "spells_learned": [ [ "cliched_curse", 1 ] ]
+  },
+  {
+    "type": "mutation",
+    "id": "FIND_THE_PRIZE",
+    "name": { "str": "Find the Prize" },
+    "points": 0,
+    "visibility": 0,
+    "ugliness": 0,
+    "description": "Once per week you mentally search for a valuable location.",
+    "category": [ "HOMULLUS" ],
+    "//": "Find valuable locations.",
+    "spells_learned": [ [ "find_the_prize", 1 ] ]
+  },
+  {
+    "type": "mutation",
+    "id": "NO_TRACE_YET_I_WISH",
+    "name": { "str": "No Trace Yet I Wish" },
+    "points": 0,
+    "visibility": 0,
+    "ugliness": 0,
+    "description": "The Homullus gains the ability to stop emitting scent at will.  They can use this to control when scent tracking creatures will be able to find them.",
+    "category": [ "HOMULLUS" ],
+    "transform": {
+      "target": "NO_TRACE_YET_I_WISH_active",
+      "msg_transform": "Your body stops producing scent.",
+      "active": true,
+      "moves": 10
+    },
+    "active": true
+  },
+  {
+    "type": "mutation",
+    "id": "NO_TRACE_YET_I_WISH_active",
+    "name": { "str": "No Trace Yet I Wish (on)" },
+    "description": "You don't currently produce scent.",
+    "copy-from": "NO_TRACE_YET_I_WISH",
+    "valid": false,
+    "transform": { "target": "NO_TRACE_YET_I_WISH", "msg_transform": "Your body begins producing scent.", "active": false, "moves": 10 },
+    "flags": [ "NO_SCENT" ]
+  },
+  {
+    "type": "mutation",
+    "id": "HOMULLUS_EYES",
+    "name": { "str": "Homullus Eyes" },
+    "points": 1,
+    "visibility": 2,
+    "ugliness": 0,
+    "description": "Upon gaining this ability the Homullus's eyes begin change to look like doll's eyes  always staring at the world.  Glare no longer impacts them.",
+    "category": [ "HOMULLUS" ],
+    "integrated_armor": [ "integrated_homullus_eyes" ]
+  },
+  {
+    "type": "mutation",
+    "id": "TRUST_ME",
+    "name": { "str": "Trust Me" },
+    "points": 1,
+    "visibility": 0,
+    "ugliness": 0,
+    "description": "You are much more succesful at lying to people now.",
+    "category": [ "HOMULLUS" ],
+    "social_modifiers": { "lie": 40 },
+  },
+  {
+    "type": "mutation",
+    "id": "DEATH_IS_BUT_A_DOOR",
+    "name": { "str": "Death Is But A Door" },
+    "points": 7,
+    "visibility": 0,
+    "ugliness": 0,
+    "description": "Upon gaining this ability the Undine's eyes begin reflecting their mood with colors of the deep waters, pitch black when angry or sad and green-blue when pleased.  Glare no longer impacts them.",
```

Comment:
It seems to be a copy of the Undine eyes mutation's description, probably as a placeholder while the description for this mutation wasn't put in place, since Death is But A Door is not an eye mutation.